source/ui.py

```diff
@@ -21,13 +21,15 @@
 from comtypes import automation
 from comtypes import COMError
 from html import escape
+
+import nh3
 from logHandler import log
 import gui
 import speech
 import braille
 from config.configFlags import TetherTo
 import globalVars
-from typing import Optional
+from typing import Callable, Optional
```

Comment:
`typing.Callable` is deprecated.
```suggestion
from typing import Optional
from collections.abc import Callable
```